WordPress/gutenberg
# Storybook: Add back subcomponents to props table

Follow-up to #53520
Partially addresses #53541

## What?

Add back all the subcomponents to the props tables.

## Why?

Storybook 7 does not support the display of subcomponents in the props tables, so they had to be removed.

This PR returns the props tables to what they used to look like, by using the deprecated (but still available) `ArgsTable` block (https://github.com/storybookjs/storybook/issues/20782) instead of the new default `Controls` block. It is a bandaid fix while we work on documenting the subcomponents in more appropriate ways (#53541).

## How?

Using a custom Docs page structure so we can use the old `ArgsTable` component.

## Testing Instructions

1. `npm run storybook:dev`
2. See any of the updated components. The subcomponent tabs should be back in the props tables.

## Screenshots or screencast 

<img width="1040" alt="Props table with subcomponent tabs" src="https://github.com/WordPress/gutenberg/assets/555336/e1561006-bb2f-4dd5-9887-f2172af7bdf9">
